components/error-text/README.md

```diff
@@ -9,9 +9,4 @@ ErrorText
 
 
 
-### Properties
-Prop | Required | Default | Type | Description
-:--- | :------- | :------ | :--- | :----------
- `children` | true |  | node | 
```

Comment:
this should probably be left in, with a description that this is how one sets the text for this component